centroid_decompostition.cpp

```diff
@@ -0,0 +1,92 @@
+#include <iostream>
+#include <vector>
+
+using namespace std;
+
+class Graph 
+{
+private:
+	int ver;
+	vector<int>* adj;
+	vector<int>* centroidTree;
+	int* sizes;
+	bool* deleted;
+public:
+	Graph(int num) : ver(num) {
+		adj = new vector<int>(ver + 1);
+		centroidTree = new vector<int>(ver + 1);
+		sizes = new int[ver + 1];
+		deleted = new bool[ver + 1];	
+	}
```

Comment:
Also, use 4 spaces, not tabs

Reply:
Thank you for your thorough review, I'll make sure to correct my mistakes. And I'm sorry, I made new pull request for this (mistakenly?) but on the other pull request, the file is in proper folder.